invenio_cli/helpers/commands.py

```diff
@@ -382,3 +387,12 @@ def containerize(self, force, install):
 
         click.secho(
             'Instance running!\nVisit https://localhost', fg='green')
+
+    def demo(self):
+        """Add demo records into the instance."""
+        project_shortname = self.cli_config.get_project_shortname()
+
+        # The way of starting in container mode ensures the
+        # containers are up
```

Comment:
Minor: Probably safer and more consistent to have containers started again though...

Reply:
Shelved (as discusse IRL)